tools/copy_engine_content.py

```diff
@@ -21,8 +20,24 @@
     content_dirs = {
         os.path.realpath(os.path.join(args.unreal_engine_dir, "Samples", "StarterContent", "Content", "StarterContent")): \
             os.path.realpath(os.path.join(args.unreal_project_dir, "Content", "StarterContent")),
+        os.path.realpath(os.path.join(args.unreal_engine_dir, "Templates", "TemplateResources", "High", "LevelPrototyping", "Content")): \
+            os.path.realpath(os.path.join(args.unreal_project_dir, "Content", "LevelPrototyping")),
         os.path.realpath(os.path.join(args.unreal_engine_dir, "Templates", "TemplateResources", "High", "Characters", "Content", "Mannequins")): \
-            os.path.realpath(os.path.join(args.unreal_project_dir, "Content", "Characters", "Mannequins"))}
+            os.path.realpath(os.path.join(args.unreal_project_dir, "Content", "Characters", "Mannequins")),
+        os.path.realpath(os.path.join(args.unreal_engine_dir, "Templates", "TP_ThirdPersonBP", "Content", "ThirdPerson")): \
+            os.path.realpath(os.path.join(args.unreal_project_dir, "Content", "ThirdPerson")),
+        os.path.realpath(os.path.join(args.unreal_engine_dir, "Templates", "TP_ThirdPersonBP", "Content", "__ExternalActors__", "ThirdPerson")): \
```

Comment:
Are these `__ExternalWhatever__` folders necessary to include and copy? It looks like they have a bunch of temp files in them. Why do we need these here?